src/common/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,69 @@
+import cn from 'classnames';
+import React, {Component} from 'react';
+import ITabsProps from './Tabs.d';
+
+import styles from './tabs.scss';
+
+export default class Tabs extends Component<ITabsProps> {
+	public static defaultProps = {
+		tabs: [
+			{
+				title: '1',
+				content: 'content',
+			},
+		],
+		tabsHeight: 50,
+	};
+
+	public state = {
+		currentTab: 0,
+	};
+
+	public tabStyle = (index: number) => {
```

Comment:
do not make these style objects inside a component. Use "cn" from "classnames" instead. Move related styles to a someStyle.scss file and import if required